src/rrdtool.c

```diff
@@ -885,7 +871,7 @@ static int rrd_config(const char *key, const char *value) {
             "be greater than 0.\n");
       return 1;
     }
-    cache_flush_timeout = tmp;
+    cache_flush_timeout = TIME_T_TO_CDTIME_T(tmp);
```

Comment:
I'd prefer to copy the way `CacheTimeout` is handled and use a double, `atof()` and `DOUBLE_TO_CDTIME_T()` here.